indico/filters/submission.py

```diff
@@ -0,0 +1,5 @@
+from .base import BaseFilter
+
+
+class SubmissionFilter(BaseFilter):
```

Comment:
if implemented class filters won't get more complicated than this, up to you but I think placing this class, and future BaseFilter subclasses into `indico/filters/__init__.py` could also work? this is totally up to you tho